core/modules/layout/js/grid-fallback.js

```diff
@@ -17,7 +17,9 @@
 Backdrop.behaviors.gridFloatFallback = {
   attach: function(context) {
     if (!Backdrop.featureDetect.flexbox()) {
```

Comment:
Safari does have flexbox, right? I think this should be an OR not an if inside an if.